lib/shopify_cli/theme/dev_server_common.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require "pathname"
+
+module ShopifyCLI
+  module Theme
+    module DevServer
+      # Errors
+      Error = Class.new(StandardError)
+      AddressBindingError = Class.new(Error)
+
+      class DevServerCommon
```

Comment:
I've left a [comment](https://github.com/Shopify/shopify-cli/pull/2429#pullrequestreview-1030063582) that would allow the `DevServer` name. But, please, let me know wdyt :)